query_stats_agg_idc.txt

```diff
@@ -0,0 +1,48 @@
+WITH temp_query as (
+SELECT
+  DISTINCT
+  SeriesInstanceUID,
+  StudyInstanceUID,
+  PatientID,
+  RepetitionTime,
+  EchoTime,
+  SliceThickness,
+  MagneticFieldStrength,
+  PixelSpacing[SAFE_OFFSET(0)] as PixelSpacing,
+  CASE
+    WHEN SeriesDescription LIKE '%t2_tse_tra%' AND collection_id = "prostatex" AND Modality = "MR" THEN "t2w_prostateX" 
```

Comment:
I would suggest to have a column that just defines the contrast of the series (e.g., "T2W"), and not mix contrast type with the collection ID. We already have collection ID in an existing column.

Reply:
ok, i will separate collection_id and SeriesDescription, then group by colllection_id, SeriesDescription.